drivers/gpu/drm/vc4/vc4_hdmi.c

```diff
@@ -1128,6 +1189,11 @@ static int vc4_hdmi_encoder_atomic_check(struct drm_encoder *encoder,
 	if (mode->flags & DRM_MODE_FLAG_DBLCLK)
 		pixel_rate *= 2;
 
+	if (conn_state->max_bpc == 12)
+		pixel_rate *= 150 / 100;
```

Comment:
this looks odd - doesn't 150/100 just evaluate to 1, so that line is a no-op?

same thing 2 lines below

Reply:
Quite right.
I've just pulled them in off the dri-devel mailing list - they haven't been reviewed yet. Feel free to comment in your own right.

Reply:
Yep...

Reply:
I'm emailing Maxime with a heads up on this one.

Reply:
thanks!